use/extending.md

```diff
@@ -61,7 +62,8 @@ so there are several ways to install binary R packages on Linux.
 #### [RStudio Public Package Manager](https://packagemanager.rstudio.com/)
 
 RStudio Package Manager (RSPM) provides binary R packages for specific Linux distributions[^rspm].
-RSPM behaves like CRAN, so users can install packages just as if they were installed from CRAN.
+Since RSPM provides all packages on CRAN as a CRAN mirror,
+users can install packages just as if they were installed from CRAN.
```

Comment:
Modulo possible issues with missing dependencies.

Reply:
Of course that needs to be explained here!
I thought I had explained it in the following section, but there may be a more appropriate way to write it.

https://github.com/rocker-org/website/blob/2eaef671eab75199fab5b53444c6587cfc6ff779/use/extending.md?plain=1#L81-L85

Reply:
Nice, had not seen that.  It's a complicated topic, and RSPM is a great service particularly for the wide range of OSs and distros it cover -- which makes the 'almost but not quite' nature on Linux (esp Ubuntu) more annoying.